components/NewRegistrationForm/Steps/PersonalDetailsStep/PersonalDetailsStep.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Field } from 'formik';
+import * as Yup from 'yup';
+import { validationErrorMessages } from 'common/constants/messages';
+import Input from 'components/Form/Input/Input';
+import styles from './PersonalDetailsStep.css';
+
+class PersonalDetailsSteps extends React.Component {
```

Comment:
Nitpick, but I'm assuming you didn't mean to pluralize just this step.

Reply:
I didn't! Good catch.